tests/src/shader-lab/ShaderLab.test.ts

```
@@ -188,68 +182,82 @@
+    // @ts-ignore
+    shaderLabVerbose._parse(errorShader);
+    // @ts-ignore
+    expect(shaderLabVerbose._errors.length).to.eq(3);
+    // @ts-ignore
+    assert.instanceOf(shaderLabVerbose._errors[0], GSError);
+    // @ts-ignore
+    assert.instanceOf(shaderLabVerbose._errors[1], GSError);
+    // @ts-ignore
+    assert.instanceOf(shaderLabVerbose._errors[2], GSError);
+
+    debugger;
```

Biome

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)